docs/kubeadm.md

```diff
@@ -0,0 +1,210 @@
+---
+---
+
+<style>
+li>.highlighter-rouge {position:relative; top:3px;}
+</style>
+
+## Overview
+
+This quickstart will show you how to easily install a secure Kubernetes cluster on any machines running Linux, using a tool called `kubeadm` which is part of Kubernetes.
+
+This process should work with local VMs, physical servers and/or cloud servers.
+It is intended to be simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
+
+**The `kubeadm` tool is currently in alpha but please try it out and give us feedback!**
+
+### tl;dr &mdash; the really short version
+
+You will:
+
+1. Install packages on all machines
```

Comment:
This tl;dr is not meant to be followed, it's meant to give a sense of how easy the process is to encourage people to continue. Prerequisites are mentioned below.